hive/agents/qnets/sequence_models.py

```diff
@@ -0,0 +1,135 @@
+import torch
+from torch import nn
+
+from hive.utils.registry import registry, Registrable
+
+
+class SequenceModule(nn.Module, Registrable):
```

Comment:
I don't really see what the point of this class is. It's not really providing any additional logic. I think instead of this, you can just do:
```
registry.register_all(FunctionApproximator,
{
'LSTM':FunctionApproximator(torch.nn.LSTM),
'GRU':FunctionApproximator(torch.nn.GRU),
}
```

In the convrnnnetwork class, you can just have it take in a FunctionApproximator for the seqence_fn and create it directly. If you really want to differentiate SequenceModels as a type, you should just have it subclass Registrable and provide a type name, similar to what FunctionApproximator does. It shouldn't subclass nn.Module.

Reply:
Although `SequenceModule` is simply a wrapper for LSTM or GRU for now, we made it a separate class mainly because we can implement our own recurrent module later on.

Reply:
done